README.md

```diff
@@ -46,4 +47,19 @@ Download the latest .tgz release and upload into *Modules* > *Module Manager* >
 
 Go to the *Manage* > *Services* tab in the PAY. Admin Panel to enable extra payment methods
 
+#### Update instructions
+
+1. Uninstall the old version
+2. install the newest version of the plugin.
+
+# Usage
+
+**More information on this plugin can be found on https://docs.pay.nl/**
```

Comment:
Hier is het handig te linken naar de Magento link van de docs ipv de main pagina van de docs